test/integration/targets/include_parent_role_vars/tasks/main.yml

```diff
@@ -0,0 +1,2 @@
+# Copyright 2019 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
Why add an empty file?  Does something fail without it?

Reply:
I was under the impression that at least one */main.yml would have to exist for it to be considered an actual role.

That said, I will adjust it as I had considered a use case that needs to be tested that is currently not yet tested.